### PR TITLE
elections: Kick the March 2019 election off

### PR DESCRIPTION
**Pull request by sameo**
2 seats for elections, process starts in 2 weeks.

Fixes: #74

Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>

### CODE REVIEW

**Comment by sameo** [comment]
cc @egernst @WeiZhang555 @gnawux @jon

**Comment by sameo** [comment]
@sboeuf @ClaireMaissey

File: elections/arch-committee-2019-03/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Architecture Committee Elections: September 2018
```
Comment (grahamwhaley) [inline elections/arch-committee-2019-03/README.md]: not Sept 2018 - Feb 2019?

Comment (sameo) [reply elections/arch-committee-2019-03/README.md]: Fixed, sorry.

**Comment by sameo** [comment]
> Should this be for two seats this time around?

Indeed. Copy paste fail.
Fixed now, thanks.

**Comment by grahamwhaley** [comment]
@ttx - I see the Zuul CI checks have passed at https://zuul.opendev.org/t/kata-containers/builds, but don't see their status reflected above. Any ideas?

**Comment by grahamwhaley** [comment]
ah, there they are.... merging....

**Comment by raravena80** [comment]
> /cc @grahamwhaley (me) and I believe @mimeriza as the volunteers to help drive the process.

I can help with this too. Let me know.

**Pull request merged**